src/Components/Scenes/RequestAssets/MakeRequests/MakeRequests.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory } from 'react-router-dom';
+import SnackBar from '../../../Presentational/RequestAssets/SnackBar';
+
+const tableStyles = makeStyles(theme => ({
+  button: {
+    textTransform: 'none',
+    fontWeight: 'bold',
+    color: theme.palette.common.white,
+    margin: 8
+  }
+}));
+
+const MakeRequests = props => {
+  var {
+    makeRequestsFetch,
+    notificationMessage,
+    isLoading,
+    selectedEmployees,
+    selectedSearchResultCopy,
+    justification
+  } = props;
+  const classes = tableStyles();
+  let history = useHistory();
+
+  var [notification, setNotification] = React.useState(false);
+
+  const handleOpenNotification = () => {
+    setNotification(true);
+  };
+
+  const handleCloseNotification = () => {
+    setNotification(false);
+  };
+  return (
+    <div className={classes.buttonStyle}>
+      <Button
+        className={classes.button}
+        variant="contained"
+        color="primary"
+        disabled={
+          !selectedSearchResultCopy > 0 ||
```

Comment:
should these be `&&` vs `||`? I'm under the assumption that you need a justification, selected assets, and employees selected to request assets.

Reply:
My logic in here is checking any of them is `false` it will make disabled the button